pkg/deploy/deploy.go

```diff
@@ -384,9 +389,9 @@ func SubscriptionExists(cli client.Client, namespace string, name string) (bool,
 	return true, nil
 }
 
-// Checks if an Operator with operatorprefix is installed in the given namespace list
-// TODO: if we need to check exact verison of the operator installed, can append vX.Y.Z later
+// OperatorExists checks if an Operator with 'operatorprefix' is installed in the given namespace.
```

Comment:
```suggestion
// OperatorExists checks if an Operator with 'operatorprefix' is installed.
```
my previous comment was wrong, it does not require a given namespace.

Reply:
You can ```suggest and we can get it fixed as part of this pr

Reply:
haha thought i made a suggestion, by removing "in the given namespace."